posts/2021-08-31-open-liberty-starter.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Get started using the Open Liberty Starter"
```

Comment:
I think we can improve on the title.... maybe "Introducing the Open Liberty Starter!" or something else.... can you ask @dmuelle or @gcharters for suggestions?

Reply:
@dmuelle and @gcharters, do you have any advice on the title?